foursight_core/checks/ecs_checks.py

```diff
@@ -0,0 +1,68 @@
+from dcicutils.ecs_utils import ECSUtils
+from .helpers.confchecks import *
+
+
+@check_function()
+def ecs_status(connection, **kwargs):
+    """ ECS Status check reports metadata on the Clusters/Services running on
+        ECS in the account where foursight has been orchestrated.
+    """
+    check = CheckResult(connection, 'ecs_status')
+    full_output = {
+        'ECSMeta': {
+            'clusters': {}
+        }
+    }
+    client = ECSUtils()
+    cluster_arns = client.list_ecs_clusters()
+    for cluster_arn in cluster_arns:
+        if 'cgap' in cluster_arn:
```

Comment:
One of the few things in Python I consider a horrible failure: is this doing a list membership test or a substring test. I wonder if we should have functions to clarify that? I guess here if you know an ARN is a string it's supposed to be intelligible, but just barely. I might still like a separate function that separates out this heuristic so you can do `is_cgap_arn(arn)`.